joomla/joomla-cms
# [4.0] Blocked user logout action should not be Logged when user is not logged in Frontend

### Summary of Changes
If user is blocked , Logout activity should not be logged.

### Testing Instructions
Login to Joomla Backend with Superuser access

Go to Users => Manage
Disable any user
Go to Users => Users Action Log
See the latest activity. "User vjtest logged out from admin"

### Expected result

The logout entry for blocked user should not be logged when user is not logged in Front end, It should only log user is updated by Super user.

### Actual result

There were two activities 
1. For update of user by super user.
2. User is logged out from system.

![screenshot-vijayktest joomla com-2019 10 19-17_27_56](https://user-images.githubusercontent.com/17097405/67144636-4d746280-f296-11e9-89e2-275884140d48.png)
![screenshot-vijayktest joomla com-2019 10 19-17_26_53 (1)](https://user-images.githubusercontent.com/17097405/67144637-4d746280-f296-11e9-88cb-5a8c9dfd52aa.png)


### Documentation Changes Required
No
